src/asus-stylus

```diff
@@ -0,0 +1,3 @@
+#!/bin/bash
+
+python3 /usr/lib/asus-stylus/asus-stylus.py $@
```

Comment:
@SelfRef I would prefer to have a specific `python3` instance (virtual environment) for this package only with a defined version of `libevdev`


something similar is already done in [asus-numberpad-driver](https://github.com/asus-linux-drivers/asus-numberpad-driver)

install script
```
   sudo apt ... python3-dev python3-virtualenv  ...

   # create Python3 virtual environment
    virtualenv --python=$(which python3) /usr/share/asus-numberpad-driver/.env
    source /usr/share/asus-numberpad-driver/.env/bin/activate
    pip3 install --upgrade pip
    pip3 install --upgrade setuptools
    pip3 install -r requirements.txt
    deactivate
```

systemd service
```
/usr/share/asus-numberpad-driver/.env/bin/python3 <driver-python-code>
```

Reply:
I don't know what about other distros, but in Arch that's not a correct way of packaging. A Python program should only use system-wide Python packages. If some is missing or the specific version is needed for compatibility - it can be packaged separately.

A program that does not use system packages (so it's binary with dependencies) can be installed in `/opt/` but that's a method meant mostly for binary releases only like Discord.

If you already added virtual env in other project, then you can easily do the same here as you like, but it wasn't my intention to add it in this PR.

Nevertheless, I strongly advise against using venv in native distro package itself or at least check other's distro guidelines what they say about it.

Reply:
Done :+1:

Reply:
I wonder how other distros handle `/usr/bin/` and `/bin/`. In Arch it's just a symlink, in BSD they are separate but Linuxes probably almost always symlink these two dirs.